Shared/Sources/ColourItem.swift

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import UIKit
+
+struct ColourItem {
+  let name: String
+  let color: UIColor
+
+  init(name: String, color: UIColor) {
+    self.name = name
+    self.color = color
+  }
+}
+
+extension ColourItem {
+  private var rgbaText: String {
+    let ciColor = CIColor(cgColor: color.cgColor)
```

Comment:
👍 Good on you for finding the solution to extract RGB value.

Today I also learn`CIColor`.